xml/art_raspberry-pi.xml

```diff
@@ -1204,26 +1077,25 @@ Activating sle-module-desktop-applications &product-ga;.&product-sp; aarch64 ...
 Successfully registered system
 </screen>
    <para>
-    You can then install the <emphasis role="italic">BlueZ</emphasis> software
+    Install then the <emphasis role="italic">BlueZ</emphasis> software
     stack:
    </para>
    <screen>&prompt.root;zypper install bluez</screen>
    <para>
-    To enable the &btreg; controller for use with
-    <command>bluetoothctl</command> and related applications,
-    for example, on &rpi3; Model&nbsp;B with the default UART configuration, run:
+    Run the command below to enable the &btreg; controller for use with
+    <command>bluetoothctl</command> and related applications (for example, on
+    &rpi3; Model&nbsp;B with the default UART configuration).
```

Comment:
Please note that your tidying of things that no longer apply to SP3 conflicts with my pending rpi-x11 changes for SP2 (that I don't see integrated here), making any backports impossible through this huge Git commit. (hciattach below no longer exists in SP3.)

Reply:
Duly noted! I'd most likely to do backporting manually anyway.